dozer/cogs/_utils.py

```diff
@@ -230,7 +253,9 @@ def chunk(iterable, size):
 
 
 def bot_has_permissions(**required):
+    """Checks if bot has permissions"""
```

Comment:
More specifically, this doesn't actually test if the bot has permissions, but returns a check that does so when added to a command.